package/camera-control/camera-ctl

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+mount -o remount,rw /boot || exit 1
+
+/opt/camera-control/camera-ctl -c /boot/camera.txt -v /dev/video0
```

Comment:
This opens a bigger hole than needed for the boot to be vulnerable. Something like this will open the hole for it for only the bare minimum time needed...

```bash
#!/bin/sh

cp /boot/camera.txt /tmp/camera.txt || touch /tmp/camera.txt
/opt/camera-control/camera-ctl -c /tmp/camera.txt -v /dev/video0

mount -o remount,rw /boot || echo "Cannot remount boot as read-write, please inspect and transfer /tmp/camera.txt to /boot manually" && exit 1
cp /tmp/camera.txt /boot/camera.txt
sync
mount -o remount,ro /boot || echo "Warning: Cannot remount boot as read-only."
```

Reply:
Agree